html/changelogs/astralenigma-petcollarfix.yml

```diff
@@ -0,0 +1,6 @@
+	author: Astralenigma
```

Comment:
incorrect indentation. Also, I don't think a changelog is necessary, we usually only do changelog for bugfixes if the bug is prominent.